docker/cuda/Dockerfile

```diff
@@ -28,14 +28,14 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # Install PyTorch
 ARG TORCH_CUDA=cu124
-ARG TORCH_VERSION=stable
+ARG TORCH_RELEASE_TYPE=stable
 
-RUN if [ "${TORCH_VERSION}" = "stable" ]; then \
+RUN if [ "${TORCH_RELEASE_TYPE}" = "stable" ]; then \
     pip install --no-cache-dir torch torchvision torchaudio --index-url https://download.pytorch.org/whl/${TORCH_CUDA} ; \
-elif [ "${TORCH_VERSION}" = "nighly" ]; then \
+elif [ "${TORCH_RELEASE_TYPE}" = "nighly" ]; then \
     pip install --no-cache-dir --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/${TORCH_CUDA} ; \
 else \
-    pip install --no-cache-dir torch==${TORCH_VERSION} torchvision torchaudio --index-url https://download.pytorch.org/whl/${TORCH_CUDA} ; \
+    echo "Error: Invalid TORCH_RELEASE_TYPE. Must be 'stable' or 'nightly'." && exit 1 ; \
```

Comment:
why ? what if we want to pin a specific version ?

Reply:
I added back the TORCH_VERSION installation. I made the change because stable is not a proper TORCH_VERSION. pip install torch==stable is not correct. This interfered with the installation of the quantization packages. I now use TORCH_RELEASE_TYPE to define if we want the latest stable version or nightly.

Reply:
okay works, as long as it's configurable